File: pc.py
import re


def text_regex(reg:str, input:str):
    val = re.search(reg,input)
    return val


if __name__ == "__main__":
    reg = "/(\\.|[^[/\\\n]|\[(\\.|[^\]\\\n])*])+/([gim]+\b|\B);"
    print("Regex:", reg)
    string = "/\s+/;"
    print("Input String:", string)
    print("Result", text_regex(reg,string))

File: PC.java
import java.util.regex.Matcher;
import java.util.regex.Pattern;

public class PC {
    public static void main(String[] args) {
        String reg = "/(\\.|[^[/\\\n]|\\[(\\.|[^\\]\\\n])*])+/([gim]+\\b|\\B);";
        System.out.println("Regex: " + reg);
        String input = "/\\s+/;";
        System.out.println("Input: " + input);
        Pattern pattern = Pattern.compile(reg, Pattern.CASE_INSENSITIVE);
        Matcher matcher = pattern.matcher(input);
        System.out.println("Result: "+ matcher.find());

    }
}
